lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1530,4 +1542,18 @@ void SessionState::update_bearer_deletion_req(
   update.delete_req.mutable_eps_bearer_ids()->Add(
       bearer_id_by_policy_[PolicyID(policy_type, rule_id)]);
 }
+
+void SessionState::update_data_usage_metrics(
+    uint64_t bytes_tx, uint64_t bytes_rx) {
+  auto sid    = get_config().common_context.sid().id();
```

Comment:
nit: `const auto&` ?

Reply:
added `const` as getting the reference can lead to memory seg fault caused by a dangling reference